src/main/java/com/custom/fuelsdk/PaginationETSoapObject.java

```diff
@@ -0,0 +1,365 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.custom.fuelsdk;
+
+import com.exacttarget.fuelsdk.ETClient;
+import com.exacttarget.fuelsdk.ETDataExtensionColumn;
+import com.exacttarget.fuelsdk.ETDataExtensionRow;
+import com.exacttarget.fuelsdk.ETExpression;
+import com.exacttarget.fuelsdk.ETFilter;
+import com.exacttarget.fuelsdk.ETResponse;
+import com.exacttarget.fuelsdk.ETResult;
+import com.exacttarget.fuelsdk.ETSdkException;
+import com.exacttarget.fuelsdk.ETSoapConnection;
+import com.exacttarget.fuelsdk.ETSoapObject;
+import com.exacttarget.fuelsdk.annotations.SoapObject;
+import com.exacttarget.fuelsdk.internal.APIObject;
+import com.exacttarget.fuelsdk.internal.RetrieveRequest;
+import com.exacttarget.fuelsdk.internal.RetrieveRequestMsg;
+import com.exacttarget.fuelsdk.internal.RetrieveResponseMsg;
+import com.exacttarget.fuelsdk.internal.Soap;
+import org.apache.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.exacttarget.fuelsdk.ETDataExtension.retrieveColumns;
+
+/**
+ * Extending class to fix the bug with Fuel-SDK on pagination.
```

Comment:
what's the pagination bug how was it fixed? can you add some details in the comemnts

Reply:
Updated with additional details.